packages/backend/src/queryBuilder.ts

```diff
@@ -245,11 +245,23 @@ export const buildQuery = ({
         return undefined;
     };
 
+    const tablesSqlWhere = [...joinedTables].reduce<string[]>(
+        (acc, tableName) => {
+            const table = explore.tables[tableName];
+
+            if (table.sqlWhere) {
+                return [...acc, table.sqlWhere];
+            }
+            return acc;
+        },
+        [],
+    );
+
     const nestedFilterSql = getNestedFilterSQLFromGroup(filters.dimensions);
+    const nestedFilterWhere = nestedFilterSql ? [nestedFilterSql] : [];
+    const allSqlFilters = [...tablesSqlWhere, ...nestedFilterWhere];
     const sqlWhere =
-        filters.dimensions !== undefined && nestedFilterSql
-            ? `WHERE ${nestedFilterSql}`
-            : '';
+        allSqlFilters.length > 0 ? `WHERE ${allSqlFilters.join(' AND')}` : '';
```

Comment:
I tried to avoid doing `WHERE true` because I'm not sure if that will be compatible with all warehouses

Reply:
It works! 
- https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types
- https://docs.databricks.com/sql/language-manual/data-types/boolean-type.html
- https://docs.snowflake.com/en/sql-reference/data-types-logical
- https://www.postgresql.org/docs/current/datatype-boolean.html
- https://trino.io/docs/current/language/types.html#boolean

Reply:
do you still prefer using `WHERE true`

Reply:
Good point, it's done now, in future we could change this if we're adding more and more filters